test/integration/scheduler_perf/scheduler_perf_types.go

```diff
@@ -0,0 +1,30 @@
+package benchmark
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+		"k8s.io/kubernetes/pkg/api/v1"
+)
+
+// High Level Configuration for all predicates and priorities.
+type schedulerPerfConfig struct {
+	NodeAffinity nodeAffinity
+	InterpodAffinity interpodAffinity
+}
+
+// interpodAffinity priority configuration details.
+type interpodAffinity struct {
+	Enabled     bool
+	Operator    metav1.LabelSelectorOperator
```

Comment:
Even for pod affinity, we need a label selector. So I guess, we need operator. I was looking at https://raw.githubusercontent.com/kubernetes/kubernetes.github.io/master/docs/concepts/configuration/pod-with-pod-affinity.yaml. Please let me know, if it is not needed.